### PR TITLE
Fix for: Copy/Paste works incorrectly

### PR DESCRIPTION
**Pull request by reggie7**
Fix for #702

### CODE REVIEW

**Review by ehdatheadwire — APPROVED** [review]
Checked locally against develop-sling12 and confirmed that the fix/changes have been made. Looks good!

Screen Shot 2021-01-12 at 2 03 49 PM

File: admin-base/ui.apps/src/main/js/apiImpl.js [hunk 1]
```diff
@@ -1121,7 +1121,7 @@ class PerAdminImpl {
     logger.fine(arguments)
     let formData = new FormData();
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/js/apiImpl.js]: if you are already removing semicolons remove this one too :D

**Review by flx-sta — CHANGES_REQUESTED** [review]

**Review by flx-sta — APPROVED** [review]

**Review by flx-sta — APPROVED** [review]
image
image
image
[Test coverage](https://github.com/Felix-Puetz/pcms-testing/commit/84943454838e829e67348d09b635bd7dccbe9444) added

**Comment by flx-sta** [comment]
Build of develop-sling12 after merge successful
image

**Pull request merged**